src/administration/components/AdminPanel.jsx

```diff
@@ -117,6 +112,9 @@ class Admin extends Component {
                             </div>
                         </div>
                     </form>
+
+
+                    <ValidationForm addVolunteerFunc={this.handleAddVolunteer}/>
```

Comment:
I'll resolve conflicts, but this function call is still not working properly. You get redirected to the 'home page' after you add a volunteer... @coka